packages/update-check/package.json

```diff
@@ -22,21 +22,26 @@
 	],
 	"scripts": {
 		"build": "tsc --project tsconfig.prod.json",
-		"dev": "tsc --watch"
+		"dev": "tsc --watch",
+		"test": "vitest --watch=false"
 	},
 	"dependencies": {
-		"boxen": "7.1.0",
-		"chalk": "5.2.0",
-		"find-cache-dir": "4.0.0",
-		"node-fetch-cache": "3.1.3",
-		"semver": "7.5.1"
+		"boxen": "7.1.1",
+		"chalk": "5.3.0",
+		"filenamify": "6.0.0",
+		"find-cache-dir": "5.0.0",
+		"got": "13.0.0",
```

Comment:
I am still wondering if we really need to introduce a new dependency (with its subdependencies) here for this task 🤔 

- Much more popular than node-fetch-cache
It looks like we're loading this library (or previously node-fetch-cache) just to cache the single response from this one fetch request 😬 

- Proper [RFC 7234](https://httpwg.org/specs/rfc7234.html) HTTP caching
This proposed RFC has been mad obsolete by https://www.rfc-editor.org/rfc/rfc9111.html

Is it possible to do without this extra dep? 😬

Reply:
> It looks like we're loading this library (or previously node-fetch-cache) just to cache the single response from this one fetch request 😬

Right, but we definitively need to cache it! Otherwise the request is executed on every CLI call (the response is over a megabyte in size!).

> Is it possible to do without this extra dep? 😬

Probably, but as we know proper cache implementation is not that simple and might be a lot of work. Not sure if it is worth it in this case...

Reply:
(@br41nslug But maybe something we can tackle later? I also like to keep the number of external dependencies as low as possible 😬)

Reply:
exactly, i resolved the question and am running tests on the PR now 😄